Code/MinimalLib/common.h

```diff
@@ -144,12 +153,11 @@ RWMol *mol_from_input(const std::string &input,
     LPT_OPT_GET(makeDummiesQueries);
   }
   try {
-    if (input.find("M  END") != std::string::npos) {
+    if (haveMolBlock) {
       bool strictParsing = false;
       LPT_OPT_GET(strictParsing);
-      res = MolBlockToMol(input, false, removeHs, strictParsing);
-    } else if (input.find("commonchem") != std::string::npos ||
-               input.find("rdkitjson") != std::string::npos) {
+      res = MolBlockToMol(input, false, false, strictParsing);
```

Comment:
Could you add a comment on the rationale for not sanitizing at the Smiles or Mol block parser level, I'm a bit confused by this logic.

Reply:
Done. The reason is that MinimalLib is primarily used for molecule visualization, and in general you want to be able to visualize also molecules which will fail sanitization (i.e., because they fail kekulization), so we have a rather fine-grained control on sanitization options, which is why we skip the initial default sanitization.